install

```diff
@@ -4,15 +4,20 @@
 # anywhere you like.
 HOMEBREW_PREFIX = "/usr/local".freeze
 HOMEBREW_REPOSITORY = "/usr/local/Homebrew".freeze
+HOMEBREW_CORE_TAP = "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core".freeze
 HOMEBREW_CACHE = "#{ENV["HOME"]}/Library/Caches/Homebrew".freeze
 HOMEBREW_OLD_CACHE = "/Library/Caches/Homebrew".freeze
 BREW_REPO = "https://github.com/Homebrew/brew".freeze
+BREW_REPO_INSECURE = "git://github.com/Homebrew/brew.git".freeze
 CORE_TAP_REPO = "https://github.com/Homebrew/homebrew-core".freeze
+CORE_TAP_REPO_INSECURE = "git://github.com/Homebrew/homebrew-core.git".freeze
 
 # TODO: bump version when new macOS is released
 MACOS_LATEST_SUPPORTED = "10.14".freeze
 # TODO: bump version when new macOS is released
 MACOS_OLDEST_SUPPORTED = "10.12".freeze
+# TODO: keep in sync with https://github.com/Homebrew/homebrew-portable-ruby/releases/latest
+PORTABLE_RUBY_INSECURE = "http://homebrew.bintray.com/bottles-portable-ruby/portable-ruby-2.3.7.leopard_64.bottle.tar.gz".freeze
```

Comment:
This maybe warrants a Homebrew/brew change instead (where we fall back to HTTP on old versions)? Going to be a bit of a nightmare keeping this up-to-date.

Reply:
Replaced with setting `HOMEBREW_BOTTLE_DOMAIN` instead.